tf_encrypted/keras/engine/base_layer.py

```diff
@@ -114,7 +116,7 @@ def add_weight(self, variable, make_private=True):
     return variable
 
   def set_weights(self, weights, sess=None):
-    """ Sets the weights of the layer.
+    """Sets the weights of the layer.
```

Comment:
According to Google style guide, this should be imperative "Set" and there should be a newline at the end of this line, I think?

Reply:
Believe you're right, let's fix in another issue